tox.ini

```diff
@@ -3,6 +3,10 @@ skipsdist=True
 envlist = unit
```

Comment:
Can you add here all the default targets for the pipeline to execute?
`envlist = lint, unit, static, coverage-report`